api/nodes/views.py

```diff
@@ -360,7 +364,10 @@ class NodeDetail(generics.RetrieveUpdateDestroyAPIView, NodeMixin):
 
     # overrides RetrieveUpdateDestroyAPIView
     def get_object(self):
-        return self.get_node()
+        node = self.get_node()
+        if node.is_registration:
```

Comment:
I'm not sure this is the desired behavior. A 404 isn't appropriate because the node does exist. However, I'm not sure that we should even return an error if requesting a registration from `/nodes/<id>/`. Registrations are nodes, after all. Thoughts, @brianjgeiger ?